plugins/Morpheus/templates/contentBlock.twig

```diff
@@ -2,8 +2,8 @@
     <div class="card-content">
         {% if title is defined and title %}
             <h2 class="card-title"
-                  {% if rate is defined and rate and rate is true %}piwik-enriched-headline="{{ title|e('html_attr') }}"
-                  {% elseif rate is defined and rate and rate %}piwik-enriched-headline="{{ rate|e('html_attr') }}"{% endif %}
+                  {% if rate is defined and rate and rate is true %}vue-entry="CoreHome.EnrichedHeadline"
+                  {% elseif rate is defined and rate and rate %}vue-entry="CoreHome.EnrichedHeadline"{% endif %}
```

Comment:
Does this change make sense? Before it was either assigned `title` or `rate` to the headline. While I'm not sure if the old angular component was handling that at all, I don't think the new vue component does, or am I wrong?
Also the second condition has a duplicate `and rate`, which is useless.

Reply:
The old directive doesn't handle a piwikEnrichedHeadline attribute value so I don't think this code actually did anything before. So my reasoning is it should be fine to remove it.